resources/build/min-ver.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+#
+# Usage: min-ver.sh
+#
+# Parses minimum-versions.inc.sh to update template file minimum-versions.md.in.
+# If minimum-versions.md is changed, commit and push changes
+#
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+. "${THIS_SCRIPT%/*}/minimum-versions.inc.sh"
+
+function generate_table {
+  echo ""
+  echo "## minimum-versions.inc.sh Variables"
+  echo ""
+  echo "|          KEYMAN Variable          |     Value    |"
+  echo "|-----------------------------------|--------------|"
+
+  # Get all the shell variables named KEYMAN*VERSION
+  strarr=( $(compgen -v KEYMAN | grep VERSION) )
```

Comment:
I found `compgen` can generate a list of shell variables starting with "KEYMAN". Is it available on Windows Git Bash?

Reply:
yes

![image](https://github.com/keymanapp/keyman/assets/4498365/fe63e4a0-6bcb-472f-9440-03b29e570f13)